handlers/handlers.go

```diff
@@ -81,9 +81,6 @@ func GetAreaViewHandler(w http.ResponseWriter, req *http.Request, ctx context.Co
 	wg.Wait()
 	//  View logic
 	basePage := c.Render.NewBasePageModel()
-	ffs := mapper.FeatureFlags{
-		EnabledBreadcrumbs: cfg.EnabledBreadcrumbs,
-	}
-	model := mapper.CreateAreaPage(basePage, areaData, relationsData, ancestorData, ffs)
+	model := mapper.CreateAreaPage(basePage, areaData, relationsData, ancestorData)
```

Comment:
Has this page been tested in Welsh? (Aware English translations will appear but can edit some of the Welsh temporarily to check that it is loading from the cy.toml) but I don't think `lang` has been used or passed to the templates unless I am missing something?

Reply:
Also can see logs, but can see they have capital letters, believe the advice for GO is to avoid capital letters and punctuation

Reply:
Thanks Dan, i sent you a message regarding these